src/geode/model/helpers/convert_brep_section.cpp

```diff
@@ -59,10 +68,310 @@ namespace
+        }
+        geode::uuid section_slice1_brep_mapping(
+            const geode::ComponentType& brep_component_type,
+            geode::uuid section_component )
+        {
+            return (
+                slice1_brep_mapping_.at( brep_component_type )
+                    .in2out( section_slice1_mapping_.at( brep_component_type )
+                                 .in2out( section_component ) ) );
+        }
+
+        void create_lines()
+        {
+            for( const auto& corner : section_.corners() )
+            {
+                extrude_corner( corner );
+            }
+        }
+        geode::uuid extrude_corner( const geode::Corner2D& section_corner )
+        {
+            const auto& corner_slice0 =
+                brep_.corner( section_slice0_brep_mapping(
+                    geode::Corner3D::component_type_static(),
+                    section_corner.id() ) );
+            const auto& corner_slice1 =
+                brep_.corner( section_slice1_brep_mapping(
+                    geode::Corner3D::component_type_static(),
+                    section_corner.id() ) );
+            const auto& line = brep_.line( brep_builder_.add_line() );
+            brep_builder_.add_corner_line_boundary_relationship(
+                corner_slice0, line );
+            brep_builder_.add_corner_line_boundary_relationship(
+                corner_slice1, line );
+            auto line_builder = brep_builder_.line_mesh_builder( line.id() );
+            const auto corner_slice0_vid =
+                line_builder->create_point( corner_slice0.mesh().point( 0 ) );
+            brep_builder_.set_unique_vertex(
+                { line.component_id(), corner_slice0_vid },
+                brep_.unique_vertex( { corner_slice0.component_id(), 0 } ) );
+            const auto corner_slice1_vid =
+                line_builder->create_point( corner_slice1.mesh().point( 0 ) );
+            brep_builder_.set_unique_vertex(
+                { line.component_id(), corner_slice1_vid },
+                brep_.unique_vertex( { corner_slice1.component_id(), 0 } ) );
+            line_builder->create_edge( corner_slice0_vid, corner_slice1_vid );
+            return line.id();
+        }
+        void create_surfaces()
+        {
+            for( const auto& line : section_.lines() )
+            {
+                extrude_line( line );
+            }
+        }
+        geode::uuid extrude_line( const geode::Line2D& section_line )
+        {
+            const auto& line_slice0 = brep_.line( section_slice0_brep_mapping(
+                geode::Line3D::component_type_static(), section_line.id() ) );
+            const auto& line_slice1 = brep_.line( section_slice1_brep_mapping(
+                geode::Line3D::component_type_static(), section_line.id() ) );
+            const auto& surface = brep_.surface( brep_builder_.add_surface(
+                geode::OpenGeodePolygonalSurface< 3 >::impl_name_static() ) );
```

Comment:
```suggestion
                 ) );
```
Default is polygonal.